src/lib/PaymentElement.svelte

```diff
@@ -13,8 +15,14 @@
   /** @type {import("./types").ElementsContext} */
   const { elements } = getContext('stripe')
 
+  /** @type {StripePaymentElementOptions["defaultValues"]} */
+  export let defaultValues
```

Comment:
Should this be `export let options` instead? That way all options are included?
Would be more future proof.

Reply:
Ok sure, let me update

Reply:
Committed